### ISSUE TITLE
Show extended info without sneaking

### ISSUE THREAD

**Issue by shysecre** [McJty/TheOneProbe#588]
Hello! Recently installed this mod and wondering is there any way to show all info about block without probe and sneaking?
I tried to turning on all settings that seems kinda related to that topic in both server and client side configs, but unfortunate for me I didn't find any useful settings.  

Right now i'm able to see full info only with probe in main hand. I really hope that there is a way to enable all info without sneaking and probe.

Game Ver: 1.18.2
Mod Ver: 1.18-5.1.2

**Comment by McJty** [comment]
Did you try the /top need command? Or /top config?

**Comment by shysecre** [comment]
Yes. And all of this commands doesn't allow you to enable extended mod without probe and sneak. Seems like right now we can get extended info only via sneak and probe

**Comment by shysecre** [comment]
@McJty So I think I can close this issue for now and just wait until you release a version where we can enable "extended" mod without sneaking and probe?

**Comment by markygnlg** [comment]
I've noticed in version theoneprobe-1.19-6.2.2 that the numbers 0-2 have no effect - I need to use "NORMAL" or "EXTENDED" for the setting (in theoneprobe-client.toml):

#Show horse stats setting (0 = not, 1 = always, 2 = sneak)
showHorseStatSetting = "NORMAL"

 - "EXTENDED" will show hearts but will only show health (as text) jump height, speed when crouched;
 - "NORMAL" will always show hearts, jump height, speed but shows health as text when crouched.

I'm about to test other settings with 'NORMAL/EXTENDED' instead of '0/1/2' to see if this helps with other settings that seem to not work.

EDIT: I'm also using the setting (in theoneprobe-common.toml):

#Is the probe needed to show the tooltip? 0 = no, 1 = yes, 2 = yes and clients cannot override, 3 = probe needed for extended info only
#Range: 0 ~ 3
needsProbe = 0

**Comment by tanyan-tanyan** [comment]
Regarding the comment markygnlg made above, I think the config's notes need to be updated, because I'm confused as to why values are "NORMAL" or "EXTENDED", when the notes only mention 0, 1, and 2.

**Comment by markygnlg** [comment]
I'll add that the 3rd option might be 'NOT', as per:

#Reveal monster eggs (0 = not, 1 = always, 2 = sneak)
showSilverfish = "NOT"

.. which I just now found within 'theoneprobe-client.toml'
.. but I need to add that I haven't played for a few months & today I want to try the minecolonies modpack  - so I do not know if this issue is resolved

**Comment by markygnlg** [comment]
Please ignore this 
(I'm making this in order to make GitHub send me an email, as I've forgotten this stuff - I'll delete this post as soon as I get an email)

**Comment by Dragonissa** [comment]
Still doesn't work 1.18-5.1.2
